common/components/WalletDecrypt/WalletDecrypt.tsx

```diff
@@ -171,7 +175,7 @@ export class WalletDecrypt extends Component<Props, State> {
       unlock: this.props.unlockPrivateKey,
       helpLink: `${knowledgeBaseURL}/private-keys-passwords/difference-beween-private-key-and-keystore-file.html`
     },
-    'view-only': {
+    [MiscWalletName.VIEW_ONLY]: {
```

Comment:
Purely a question: Is there any way to just do `WalletName.X` instead of secure / insecure / misc? Or is this important for typescript?

Reply:
@HenryNguyen5 ^

Reply:
We can, but it helps with typing if you group together each wallet type into its own enum. Then when you map over them TS can infer which wallet type you're referring to and return the specific wallet info instead of a union of all of them